eq-author/src/components/Validation/Date/DateValidation.js

```diff
@@ -184,30 +194,40 @@ class DateValidation extends React.Component {
         </Grid>
         <Grid>
           <AlignedColumn cols={START_COL_SIZE}>
-            <RelativePositionSelect
-              name="relativePosition"
-              value={relativePosition}
-              onChange={onChange}
-              onBlur={onUpdate}
-              data-test="relative-position-select"
-            >
-              {RELATIVE_POSITIONS.map(position => (
-                <option key={position} value={position}>
-                  {position}
-                </option>
-              ))}
-            </RelativePositionSelect>
+            {type === DATE && (
+              <>
+                <RelativePositionSelect
+                  name="relativePosition"
+                  value={relativePosition}
+                  onChange={onChange}
+                  onBlur={onUpdate}
+                  data-test="relative-position-select"
+                >
+                  {RELATIVE_POSITIONS.map(position => (
+                    <option key={position} value={position}>
+                      {position}
+                    </option>
+                  ))}
+                </RelativePositionSelect>
+              </>
+            )}
+            {type === DATE_RANGE && (
+              <EmphasisedText>{relativePosition}</EmphasisedText>
+            )}
             <PathEnd />
           </AlignedColumn>
           <Column cols={9}>
-            <ValidationPills
-              entityType={entityType}
-              onEntityTypeChange={this.handleEntityTypeChange}
-              PreviousAnswer={this.PreviousAnswer}
-              Metadata={this.Metadata}
-              Custom={this.Custom}
-              Now={this.Now}
-            />
+            {type === DATE && (
```

Comment:
I guess as Date and Date Range converge to have the same options we will remove this logic